guides/v2.1/ext-best-practices/extension-coding/security-performance-data-bp.md

```diff
@@ -51,3 +51,28 @@ For a list of discouraged low-level functions, review the [`Magento2/Sniffs/Func
 
 ### Always encrypt sensitive data or configurations
   Never store sensitive information in clear text within a resource that might be accessible to another control sphere. This type of information should be encrypted or otherwise protected.
+
+### Avoid running unneeded logic
+
+  Make sure that you never run some code that it won't be used in the next step. <br><br>
```

Comment:
Do we need `<br><br>` here? We try to use clean Markdown wherever it's possible.

Reply:
Nope, we don't. Removed them.